backends/core/inline.py

```diff
@@ -34,9 +50,53 @@ def _days_since(date, format):
   return (datetime.today() - datetime.strptime(str(date), format)).days
 
 
+def _bigquery(table_name, field_name):
+  def _get_bq_client():
+    try:
+      return _SESSION['bq_client']
```

Comment:
Why a `try` and not a `if 'bq_client' in _SESSION:`?
Just curious :)

Reply:
[EAFP](https://docs.python.org/3.6/glossary.html#term-eafp):

Easier to ask for forgiveness than permission. This common Python coding style assumes the existence of valid keys or attributes and catches exceptions if the assumption proves false. This clean and fast style is characterized by the presence of many [try](https://docs.python.org/3.6/reference/compound_stmts.html#try) and [except](https://docs.python.org/3.6/reference/compound_stmts.html#except) statements. The technique contrasts with the [LBYL](https://docs.python.org/3.6/glossary.html#term-lbyl) style common to many other languages such as C.

In this particular case — as in many similar cases — exception will be thrown only once, and [potentially many] subsequent calls will just get a once-populated value using an existing key.